judo-runtime-core-jsl-itest/src/test/java/hu/blackbelt/judo/runtime/core/jsl/entity/DefaultValueValidationTest.java

```diff
@@ -0,0 +1,151 @@
+package hu.blackbelt.judo.runtime.core.jsl.entity;
+
+/*-
+ * #%L
+ * JUDO Runtime Core JSL :: Parent
+ * %%
+ * Copyright (C) 2018 - 2022 BlackBelt Technology
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import com.google.inject.Inject;
+import com.google.inject.Module;
+import hu.blackbelt.judo.psm.generator.sdk.core.test.api.primitives.primitives.entitywithnotvalidintegerdefaultvalue.*;
+import hu.blackbelt.judo.psm.generator.sdk.core.test.api.primitives.primitives.entitywithnotvalidprecisiondefaultvalue.*;
+import hu.blackbelt.judo.psm.generator.sdk.core.test.api.primitives.primitives.entitywithnotvalidregexdefaultvalue.*;
+import hu.blackbelt.judo.psm.generator.sdk.core.test.api.primitives.primitives.entitywithnotvalidscaledefaultvalue.*;
+//import hu.blackbelt.judo.psm.generator.sdk.core.test.api.primitives.primitives.entitywithnotvalidstringdefaultvalue.*;
+import hu.blackbelt.judo.psm.generator.sdk.core.test.guice.PrimitivesDaoModules;
+import hu.blackbelt.judo.requirement.report.annotation.Requirement;
+import hu.blackbelt.judo.runtime.core.exception.ValidationException;
+import hu.blackbelt.judo.runtime.core.jsl.AbstractJslTest;
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.springframework.dao.DataIntegrityViolationException;
+
+import java.util.Optional;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+@Slf4j
+public class DefaultValueValidationTest extends AbstractJslTest {
+    @Inject
+    EntityWithNotValidIntegerDefaultValueDao entityWithNotValidIntegerDefaultValueDao;
+
+    @Inject
+    EntityWithNotValidPrecisionDefaultValueDao entityWithNotValidPrecisionDefaultValueDao;
+
+    @Inject
+    EntityWithNotValidScaleDefaultValueDao entityWithNotValidScaleDefaultValueDao;
+
+//    @Inject
+//    EntityWithNotValidStringDefaultValueDao entityWithNotValidStringDefaultValueDao;
```

Comment:
If this is commented due to a bug, please add ticket reference.

Reply:
[bdfe200](https://github.com/BlackBeltTechnology/judo-runtime-core-jsl/pull/191/commits/bdfe200f6a3608a459951711ccba6415c3ec1996)